README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Basic information
+My name is **Takuyo Ozaki** (he/him/his). I am from **Japan** and thirty years old. I live in Hydepark with my wife and son.
+
+## Family information
+My wife is working as a **paralegal** for a law firm in Chicago. My son is three years old and he is going to **daycare** at [UChicago Child Development Center.](https://child-care-preschool.brighthorizons.com/il/chicago/uchicagostonyisland) The attached below is one of my family photos taken in Florida. **We love Disney!**
+
+## Education
+I am a second-year graduate student at **Harris**. Now I am taking **study leave** from work to get the masters' degree of public policy. At my undergrad, I majored in **Law** at [the University of Tokyo](https://www.u-tokyo.ac.jp/en/index.html).
+
+## Professional experience
+I am **a government official** at [the Ministry of Health, Labor and Welfare of Japan](https://www.mhlw.go.jp/english/). My main job was **drafting legal bills** to reform the social systems. I have some experiences in making policies in the following fields.
+    - health insurance
+    - pension
+    - child abuse prevention
+    - labor relations
+
+## Characteristics/Hobby
+Actually, I am an *introvert* person. In my free time, I often watch movies or dramas on **Netflix** at home. One of my favorite dramas is ["Breaking Bad"](https://www.amc.com/shows/breaking-bad). This is a story about a high school chemistry teacher producing Methamphetamine. It's so exciting! 
```

Comment:
I like watching _Breaking Bad_ and Netflix as well, and I sometimes even procrastinate and binge-watching TVs.